simpinkscr/simple_inkscape_scripting.py

```diff
@@ -2093,12 +2093,16 @@ def effect(self):
             code += '\n'
         if self.options.program is not None:
             code += self.options.program.replace(r'\n', '\n')
+        #remove unnecessary import
+        code.replace("from simpinkscr import *", "")
```

Comment:
Where is this unnecessary `from simpinkscr import *` coming from?  And why not keep it in and remove the
```Python
from math import *
from random import *
from inkex.paths import Arc, Curve, Horz, Line, Move, Quadratic, Smooth, \
    TepidQuadratic, Vert, ZoneClose
```
that gets added at the top of all scripts?

Reply:
Originally, we need it in the script to make code intellisense recognize all the Functions brought by SimpInkScr.
It shall be a good idea to use it and remove the prelude as you suggested.

Reply:
However, this requires installation of the package as a prerequisite. 
If use the prelude, it does not require the installation of the package.
Sometimes, it might not be feasible to install a python package even under the user directory.
Do we still consider this case?

Reply:
It would indeed be inconvenient for users to have to install Simple Inkscape Scripting rather than simply drop the directory into an Inkscape extensions directory.  Let's retain the existing imports from `math`, `random`, and `inkex.paths`.

Where is `from simpinkscr import *` getting added to user scripts?  I don't see that line anywhere in the source code except in the line quoted at the top of this thread.

Reply:
`from simpinkscr import *` is getting added to the user scripts like the image below, the purpose is just to make it a valid script and let code intelligence recognize all the functions brought by simpinkscr.
![image](https://user-images.githubusercontent.com/735802/188887888-a9ec2c84-6570-4cf4-b276-04ecc164a8df.png)
This is the only case for that.

> It would indeed be inconvenient for users to have to install Simple Inkscape Scripting rather than simply drop the directory into an Inkscape extensions directory. Let's retain the existing imports from `math`, `random`, and `inkex.paths`.
> 
> Where is `from simpinkscr import *` getting added to user scripts? I don't see that line anywhere in the source code except in the line quoted at the top of this thread.

Reply:
Okay.  Thanks for the explanation.